src/main/resources/org/jenkinsci/plugins/ewm/definitions/Template/help-label.html

```diff
@@ -1,3 +1,10 @@
 <div>
-    Node's label. Will use this template configuration for all the nodes matching the given label.
+    <p>
+        When the actual Node label (in the Node configuration) matches the <i>Node label</i> in this <i>External
```

Comment:
Yes, currently it's just a single label. I didn't implement the feature for label expressions. 
I thought that in a first phase would be ok like that. And afterwards I would make it to support label expressions.